library/dash/src/main/java/com/google/android/exoplayer2/source/dash/DashMediaPeriod.java

```diff
@@ -69,7 +69,11 @@
         SequenceableLoader.Callback<ChunkSampleStream<DashChunkSource>>,
         ChunkSampleStream.ReleaseCallback<DashChunkSource> {
 
+  private enum CC_TYPE {
+    CEA608, CEA708
+  }
   private static final Pattern CEA608_SERVICE_DESCRIPTOR_REGEX = Pattern.compile("CC([1-4])=(.+)");
+  private static final Pattern CEA708_SERVICE_DESCRIPTOR_REGEX = Pattern.compile("([1-4])=lang:(\\w+)(,.+)?");
```

Comment:
What is the purpose of the `(,.+)?` part of the REGEX? Does it work exactly the same without it?

Reply:
The value can contain other information (like easy reader, ...) 

`1=lang:eng;2=lang:eng,war:1,er:1`

The regular expresion try to gets only the lang part of the property